Sources/SynchronizationManager.swift

```diff
@@ -164,12 +168,15 @@ public class SynchronizationManager: PersistenceIntegration {
         if let fetched = fetched?.first {
             persistable = fetched
         } else {
-            persistable = try! self.persistentStore.create(type: type)
-            persistable.id = entry.id
-            persistable.createdAt = entry.sys.createdAt
+            do {
+                persistable = try self.persistentStore.create(type: type)
+                persistable.id = entry.id
+                persistable.createdAt = entry.sys.createdAt
+            } catch let error {
+                fatalError("Could not create the Entry persistent store\n \(error)")
```

Comment:
Same here like in https://github.com/contentful/contentful-persistence.swift/pull/26/files#diff-6a7142dc6fe34b7d28c35b8871cad516R142